commands.py

```diff
@@ -8,6 +13,17 @@
 
 ### Handlers
 def handle_join(state: mantatail.ServerState, user: mantatail.UserConnection, args: List[str]) -> None:
+    """
+    Handles clien't command to join a channel on the server.
+    Command format: "JOIN #foo"
+
+    If the channel already exists, the user is put to the appropriate channel's
+    Dict of connected users.
+    If the channel does not exist, the channel is created by instantiating mantatail.Channel.
```

Comment:
This contains lots of details that will get outdated as the code is refactored. Please try to explain it at a higher level, without mentioning specific dicts, class names, function names, variable names, and so on. If someone cares about those, they will have to read the rest of the function anyway.

```suggestion
    If the channel already exists, the user is added to the channel.
    If the channel does not exist, a new channel is created with the user as the founder.
```

This applies to a lot of other docstrings too. I'm not going to add a suggestion for each one :)